packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Connect your SCIM API service to Okta
+excerpt: Create a SCIM integration directly within Okta, configure the SCIM options for that integration, and check the attributes and corresponding mappings.
```

Comment:
Same excerpt and content suggestions as those above:
'Create and configure SCIM integrations, and check the attributes with their corresponding mappings in the Okta console.'

Reply:
done